shared/src/validation-schemas/pagination/pagination.validation.ts

```diff
@@ -0,0 +1,11 @@
+import * as Joi from 'joi';
+
+import { UserGetAllRequestQueryDto } from '~/common/types/types';
+import { getNameOf } from '~/helpers/helpers';
+
+const pagination = Joi.object({
+  [getNameOf<UserGetAllRequestQueryDto>('count')]: Joi.number().min(1),
```

Comment:
Do not use magic numbers, same about `1` in `getAll` users query offset

Reply:
Sorry, how can I name a variable for `1` in `(page - 1) * count`? All I can think about is creating a variable `const previousPage = page - 1` and using it, but in that case also `1` is used as magic number

Reply:
You already have `DEFAULT_PAGE` constant -> constants for pagination would be in use both for BE and FE - as on frontend there would be initial values for pagination, so you could move them to shared folder and use `DEFAULT_PAGE` here